website/www/site/content/en/documentation/transforms/python/aggregation/approximatequantiles.md

```diff
@@ -17,7 +17,14 @@ limitations under the License.
 
 # ApproximateQuantiles
 
+{{< localstorage language language-py >}}
+
+{{< button-pydoc path="apache_beam.transforms.stat" class="ApproximateQuantile" >}}
+
 ## Examples
-See [Issue 19547](https://github.com/apache/beam/issues/19547) for updates.
+
+{{< playground height="700px" >}}
+{{< playground_snippet language="py" path="SDK_PYTHON_ApproximateQuantiles" show="approximatequantiles" >}}
+{{< /playground >}}
```

Comment:
I think these won't render until the next playground release. So it may make sense to split this into 2 PRs, one for now and one for once the playground piece is released after 2.56

Reply:
Ah, so the website can get upgraded within a release but playground gets redeployed with releases.

It's all a bit tangled and I'm not sure I have the time to split it apart. Would it be alright to just keep the PR as is and merge whenever it makes sense? (assuming there is such a moment)

Reply:
There's not a great single time to merge since we'd want the playground on the release branch, so we'd want to merge the playground fixes, wait a week or so for the release, then merge the website changes.

With that said, if you definitely don't have time we can merge right before the release branch is cut (I'm running the release so should be easy for me to pick up) and the website will just be broken for a bit in these code samples. It isn't ideal, but would probably still be better than not doing this at all.

Up to you on how to proceed here, I'll add this as a release blocker so I remember to merge though

Reply:
Outside of the linting failure the changes look good to me, so just let me know how you'd prefer to proceed